docs/building-for-ibm-dotcom.md

```diff
@@ -46,6 +46,20 @@ For English used in the United Kingdom, specify:
 the country identifier should be written in uppercase following usual 
 conventions.
 
+### Configuring available languages
+
+The available languages are set by the following tags within `<head>`:
+
+```html
+<link rel="alternate" hreflang="[language code]-[country code]" href="[redirect link]">
+```
+
+For example:
+
+```html
+<link rel="alternate" hreflang="en-us" href="https://www.ibm.com/us-en/">
```

Comment:
Do we want to add a couple more maybe 1 or 2 more to help adopters realize it's not just a one and done deal? Or maybe even just a comment.

Reply:
Done!